### ISSUE TITLE
this should be self

### ISSUE THREAD

**Issue by livep2000** [andersevenrud/OS.js-v2#101]
at javascript/handlers/example/handler.js
```javascript
function _save()
  {
  this.callAPI('settings', opts, function(response) 
```

should be

```javascript
function _save()
  {
  self.callAPI('settings', opts, function(response) 
```

Imre

**Comment by andersevenrud** [comment]
You're right. Looks like some changes has been overwritten. Will fix.

**Comment by MrSchism** [comment]
Alrighty.  I was just going to run a pull request.  I was waiting for input on #102 before hand, though.

Two birds with one pull and whatnot.

**Comment by andersevenrud** [comment]
@MrSchism 
Sure, you could do it :) I just closed #102
